core/js/typespecs/app_types.ts

```diff
@@ -173,39 +173,84 @@ export const datasetSchema = z
   .openapi("Dataset");
 export type Dataset = z.infer<typeof datasetSchema>;
 
+export const validRuntimes = ["node"] as const;
```

Comment:
Would prefer to define the enums as zod enums, similar to how we do elsewhere in this file. Just

```
export const runtimesSchema = z.enum(["node"]);
export type Runtime = z.infer<typeof runtimeSchema>
```

You can iterate through the list of members with `runtimesSchema.options`.

Reply:
can do